baseapp/abci.go

```diff
@@ -289,13 +293,11 @@ func (app *BaseApp) ProcessProposal(req abci.RequestProcessProposal) (resp abci.
 		panic("app.ProcessProposal is not set")
 	}
 
-	ctx := app.processProposalState.ctx.
```

Comment:
here and below: for my own understanding, why do we stop populating `VoteInfo`, `Proposer` and `ConsensusParam`?

Reply:
Those are being set inside `getContextForTx`, just a little cleanup to reuse code (although not really happy about it, as the function is intended to create a context for a single tx afaik; might roll this back for readability).